keyboards/anavi/macropad12/config.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RGBLIGHT_DEFAULT_MODE 9
```

Comment:
This shouldnt use a raw value and instead use one of the `RGBLIGHT_MODE_` values

Reply:
OK, I will change it to `#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_MOOD`. Thank you.